integration-tests/utils/aws/retrieveSecureDownloadDbRecord.ts

```diff
@@ -1,6 +1,6 @@
 import { dynamoDBClient } from './dynamoDBClient'
 import { QueryCommand } from '@aws-sdk/client-dynamodb'
-import { getEnv } from '../helpers'
+import { getEnv, pause } from '../helpers'
 export const retrieveSecureDownloadDbRecord = async (
```

Comment:
I don't think you're using this export, so just export `waitForDownloadHash`, as you have, and name the file `waitForDownloadHash`.

Reply:
Done.